s30904-JakubTreder/zajecia4/zadania0-9

```diff
@@ -0,0 +1,133 @@
+0 (rozgrzewkowe)
+Przypuscmy, ze nie mozesz utworzyc zadnego pliku na swoim koncie uczelnianym
+np. na szuflandii. Co sie moglo stac? (wymien co najmniej 3 opcje)
+Zakladamy, ze mozesz zalogowac sie w konsoli i nie masz zaleglosci w oplatach.
+
+Rozwiazanie:
+ - brak mijesca na dysku serwera. 
+ - uszkodzony jest dysk serwera.
+ - nie mamy uprawnien do zapisu. 
+
+
+1
+Jest dane takie drzewo katalogow:
+.
+├── ala
+│   ├── i
+│   │   └── as
+│   └── ma
+│       └── kota
+└── kot
+    └── ma
+        └── ale
+
+Znajdujesz sie w katalogu kota. Katalog ala jest w katalogu /tmp/ukos.
+Jak przejsc do katalogu ala za pomoca:
+ - sciezki wzglednej (relatywnej)
+ - sciezki bezwzglednej (absolutnej)
+
+
+Wykonanie:
+ - Sciezka wzgledna: cd ../..
+ - Sciezka bezwzgledna: cd /tmp/ukos/ala
+
+
+2
+Masz taki uklad katalogow, jak wyzej i ciagle jestes w katalogu kota. Jak
+utworzyc  poddrzewo katalogow jan/kowalski w katalogu ale za pomoca jednej
+komendy? Podaj rozwiazania uzywajac sciezek absolutnych i relatywnych.
+
+
+Wykonanie:
+ - Sciezka relatywna: mkdir -p ../../../kot/ma/ale/jan/kowalski 
+ - Sciezka absolutna: mkdir -p /tmp/ukos/kot/ma/ale/jan/kowalski 
+
+
+3
+Masz taki uklad katalogow, jak wyzej i ciagle jestes w katalogu kota. Jak
+przeniesc katalog ale do katalogu i uzywajac:
+ - zrodlo (sciezka wzgledna), miejsce docelowe (sciezka absolutna)
+ - zrodlo (sciezka absolutna), miejsce docelowe (sciezka wzgledna)
+
+
+Wykonanie:
+ - mv ../../../kot/ma/ale /tmp/ukos/ala/i
+ - mv /tmp/ukos/kot/ma/ale ../../i
+
+
+ 4
+Jak zamknac program, ktory nie reaguje na ctrl+c?
+
+
+Wykonanie:
+wyslac do programu sygnal KILL poleceniem kill -9 <pid programu>
+
+
+5
+Jak wypisac wszystkie pliki (i tylko pliki bez katalogow) w biezacym katalogu, 
+ktorych nazwa zaczyna się od al? 
+Tu bedzie troche opisu ode mnie na temat wyrazen regularnych.
+Materialy:
+http://www.robelle.com/smugbook/regexpr.html
+https://www.regular-expressions.info/
+
+Narzedzia do testowania wyrażen regularnych w przyzwoity sposob:
+https://regex101.com/
+https://regexr.com/
+
+
+Wykonanie: find ./  -name 'al*'
```

Comment:
Dobrze ale mało. To zwróci także katalogi. Dodaj odpowiedni filtr by zwróciło tylko pliki